### ISSUE TITLE
train for long text line detection and recognition

### ISSUE THREAD

**Issue by JimmyJuan** [MhLiao/MaskTextSpotter#69]
Hello @MhLiao , my scenario is mostly long text line included. Should I retrain the model by changing the parameter "MODEL.ROI_MASK_HEAD.POOLER_RESOLUTION_W" from 64 to wider, say 128 or 196 (with the change of MODEL.ROI_MASK_HEAD.RESOLUTION_W from 128 to 256/384).

If so, should I train a new model from scratch with new setting? 
How to do it ?

Thank you.

**Comment by MhLiao** [comment]
@JimmyJuan Yes, you need to change the mentioned setting and retrain it. If you do not have enough training data, you'd better use SynthText for pretraining.

**Comment by JimmyJuan** [comment]
@MhLiao Thank you.

I'm training a new pre-trained model using only SynthText data. But the loss at the beginning was as low as about 0.6, and then went up to 1.5, and gradually decrease. Is that normal? And for the detection branch, the scale of the characters was different from my real data, is the SynthText pretraining still effective?

**Comment by MhLiao** [comment]
@JimmyJuan The loss should be normal if it gradually decreases. It goes up to a high value because the positive proposals increase while there are almost no positive proposals at the beginning. I think that pretraining may be not as effective as expected, but it should not harm the training.

**Comment by JimmyJuan** [comment]
@MhLiao Thanks for the replying.
